src/app/public/modules/datepicker/datepicker.component.ts

```diff
@@ -91,7 +93,7 @@ export class SkyDatepickerComponent implements OnDestroy, OnInit {
     }
   }
 
-  public calendarId: string;
+  public calendarId: string = `sky-datepicker-calendar-${++nextId}`;
```

Comment:
☝️ this was missing from the prior PR, so I slipped it in here. The rest of this PR should be timepicker specific.